src/internal/packager/images/pull.go

```diff
@@ -63,7 +64,7 @@ func (i *ImgConfig) PullAll() error {
 
 	// Create the ImagePath directory
 	err := os.Mkdir(i.ImagesPath, 0755)
-	if err != nil {
+	if err != nil && !errors.Is(err, os.ErrExist) {
```

Comment:
This might be worth a warning, or double checking our workflow to make sure we are explicit on when we expect this directory to be created.

If this directory already exists when we are running `zarf package create` then there could be unintended contents in that directory that get added to the package (and the checksums since it was there during creation)

Reply:
Super unlikely, but probably worth consideration

Reply:
Hmm... You would have to inject something into that directory after zarf create since the tmp dir everything is built from does not exist until that is ran.  If there was malware on the system that wanted to watch your commands all they would have to do now is move that watch to when the image directory is created... (and you would probably want to do this anyway since it would be better to manipulate images that are being run in the cluster rather than just having another image there (and any manifests or layers would get overwritten on crane pull).

This was specifically to fix the retry logic that will try to remake the dir and balk